core/iwasm/interpreter/wasm_runtime.h

```diff
@@ -276,6 +310,9 @@ struct WASMModuleInstance {
     /* global and table info */
     uint32 global_data_size;
     uint32 table_count;
+#if WASM_ENABLE_TAGS != 0
+    uint32 tag_count;
```

Comment:
This may cause the AOT ABI change: the size of WASMModuleInstance and some fields' offsets should not be changed, or the old AOT file may be unable to run in the new runtime.  We will keep it unchanged until there is a big change. Could you move the fields added in this structure into WASMModuleInstanceExtra? We can use module_inst->e to access them.

Reply:
Agreed; updating as recommended